sklearn/tests/test_base.py

```diff
@@ -513,6 +514,42 @@ def test_regressormixin_score_multioutput():
     assert_warns_message(FutureWarning, msg, reg.score, X, y)
 
 
+def test_validate_X_n_feature_mismatch():
+    # Make sure ValueError is raised when there is a n_features mismatch
+    # between fit and predict/transform
```

Comment:
if the next PR is going to handle transform, etc, this test can also be in that other PR.

Reply:
Right thanks, I'm removing this now